src/pages/Editor.vue

```diff
@@ -553,16 +572,11 @@ export default {
         "hover:bg-primary hover:text-white hover:border-primary": !this.addItemDisabled,
       };
     },
-    currentItemType() {
```

Comment:
was this not being used?

Reply:
At one place but there was a duplicate computed property called `itemType` so used that instead